metrotranscom/doorway
# feat: add county map to the listings search modal

# Pull Request Template

## Description

- Add the Bay Area county map to the bottom of the listings search modal, for informational purposes.
- Fix a typo in the Spanish "View County Map" translation since the current text would mean a map of one county, whereas this is a map of several counties so we need the plural.

## How Can This Be Tested/Reviewed?

- Go to the listings page.
- Click on "Filters" to open the search modal.
- Scroll within the modal and you should see the county map below the county checkboxes list.

Desktop:
![image](https://github.com/metrotranscom/doorway/assets/67125998/ed6ced1b-8f46-4ed9-b4d9-354c4a653e39)

Mobile
![image](https://github.com/metrotranscom/doorway/assets/67125998/a205f71d-0e9a-4bfb-8a5d-efc7c99f453e)
